test/src/org/labkey/test/tests/targetedms/TargetedMSMAMTest.java

```diff
@@ -0,0 +1,533 @@
+import org.labkey.test.util.EscapeUtil;
+import org.labkey.test.util.Ext4Helper;
+import org.labkey.test.util.LogMethod;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.labkey.test.util.DataRegionTable.DataRegion;
+
+@Category({DailyB.class, MS2.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 8)
+public class TargetedMSMAMTest extends TargetedMSTest
+{
+    protected static final String SKY_FILE = "iRT Human+Standard Calibrate.zip";
+
+    @Test
+    public void testSteps() throws IOException, CommandException
+    {
+        setupFolder(FolderType.ExperimentMAM);
+        importData(SKY_FILE);
+
+        clickAndWait(Locator.linkContainingText("Panorama Dashboard"));
+        clickAndWait(Locator.linkContainingText(SKY_FILE));
+
+        verifyRunSummaryCountsPep(125,158,0, 160,628, 1, 0, 0);
+
+        clickAndWait(Locator.linkContainingText("PTM Report"));
+
+        assertElementPresent("Wrong modification count", Locator.xpath("//td[contains(text(), 'Carbamidomethyl Cysteine')]"), 9);
+        assertTextPresentInThisOrder("K.HDLDLICR.A", "K.YLECSALTQR.G", "R.YVDIAIPCNNK.G");
+        assertTextPresentInThisOrder("C245", "C157", "C163");
+        assertTextPresent("A_D110907_SiRT_HELA_11_nsMRM_150selected_2_30min-5-35", "A_D110907_SiRT_HELA_11_nsMRM_150selected_1_30min-5-35");
+
+        clickAndWait(Locator.linkContainingText("Peptide Map"));
+        assertTextPresentInThisOrder("11.3", "14.1", "14.8");
+        assertTextPresentInThisOrder("1501.75", "1078.50", "1547.71");
+        assertTextPresentInThisOrder("NU205", "NU205", "1433Z");
+        assertTextPresentInThisOrder("70-84", "325-333", "28-41");
+        assertTextPresentInThisOrder("K.ASTEGVAIQGQQGTR.L", "K.AQYEDIANR.S", "K.SVTEQGAELSNEER.N");
+        assertTextPresentInThisOrder("Carbamidomethyl Cysteine @ C156", "Carbamidomethyl Cysteine @ C244", "Carbamidomethyl Cysteine @ C93");
+    }
+
+    @LogMethod
+    protected void verifyAttributeGroupIdCalcs() throws IOException, CommandException
+    {
+        Connection cn = createDefaultConnection(false);
+        // Query for small molecule data
+        SelectRowsCommand smallMoleculeCommand = new SelectRowsCommand("targetedms", "generalmoleculechrominfo");
+        smallMoleculeCommand.setRequiredVersion(9.1);
+        smallMoleculeCommand.setColumns(Arrays.asList("MoleculeId/CustomIonName", "SampleFileId", "PeakCountRatio", "RetentionTime", "ModifiedAreaProportion", "SampleFileId/ReplicateId/RunId", "MoleculeId/AttributeGroupId", "PeptideId/AttributeGroupId"));
+        smallMoleculeCommand.addFilter("MoleculeId/AttributeGroupId", "C2X", Filter.Operator.EQUAL);
+        smallMoleculeCommand.addFilter("SampleFileId/SampleName", "13417_02_WAA283_3805_071514", Filter.Operator.EQUAL);
```

Comment:
does the filter get applied even when the column is not in columns list?

Reply:
I should have trimmed all of that code. It got copied over from the test class I copied, and is not used or needed in this new test. I just pushed a commit that deletes it. Sorry to make you review the full thing - the new test is really short!